lldb/test/API/tools/lldb-dap/evaluate/TestDAP_evaluate.py

```diff
@@ -60,7 +60,10 @@ def run_test_evaluate_expressions(
 
         # Expressions at breakpoint 1, which is in main
         self.assertEvaluate("var1", "20")
+        # Empty expression should equate to the previous expression.
```

Comment:
Please add a test that uses `memory read` to make sure that two different memory read invocations give different valid results

Reply:
Done